packages/next/server/next-server.ts

```diff
@@ -1379,12 +1382,23 @@ export default class Server {
       ]
     }
 
+    const { hotReloader } = this as any as DevServer
```

Comment:
Could you move this to https://github.com/vercel/next.js/blob/canary/packages/next/server/dev/next-dev-server.ts#L697?

Reply:
Oh, actually that might be a little trickier... hmm.

Reply:
You could do:

```
protected getPageLoader(): (
  distDir: string,
  pathname: string,
  serverless: boolean,
) => Promise<LoadComponentsReturnType>
```

and call override it in `next-dev-server` maybe?